oeq-ts-rest-api/test/settings.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as OEQ from '../src';
+import * as TC from './TestConfig';
+import { GeneralSetting, UISettings } from '../src/Settings';
+
+beforeAll(() => OEQ.Auth.login(TC.API_PATH, TC.USERNAME, TC.PASSWORD));
+
+afterAll(() => OEQ.Auth.logout(TC.API_PATH, true));
+
+test('Able to retrieve general settings', () =>
```

Comment:
```suggestion
test('That we're able to retrieve general settings', () =>
```
Just for consistency with above.